app/views/api/featurings/change.js.erb

```diff
@@ -1,4 +1,8 @@
+<%# If EventTarget is replaceWith'ed immediately, ajax:success and ajax:complete are not called and the spinner will stay %>
```

Comment:
I find this comment (and other similar ones) a bit confusing... 
What's the EventTarget? and why is there a reference to `replaceWith` if it's not used in the snippet?

Reply:
I struggled to write it as clear as possible, I worry I won't understand it myself in the future 😓.

[EventTarget](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget) is the Element that will receive the event.

Many remote elements will change themselves or be replaced by a `.js.erb` returned  by server. This happens after `ajax:send` but before `ajax:success` and `ajax:complete`. When the latter events are to be dispatched, the original element that called AJAX (the EventTarget) does not exist anymore and therefore they are lost.

This affects specifically to the ajax-in-progress spinner: it's always added but sometimes not removed.

`replaceWith` is the jQuery method, that I decided to remove after adding all these comments. I will update them.